### PR TITLE
Add feature and version support matrix

### PR DESCRIPTION
**Pull request by fghaas**
Start the Reference section. Provide an overview of features and versions running in Cleura Cloud.

* Explain which OpenStack and Ceph releases we're running where.
* List which features are supported in which regions.

### CODE REVIEW

File: docs/reference/versions/compliant.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Compliant Cloud
```
Comment (evrardjp) [inline docs/reference/versions/compliant.md:1]: I am not into explaining which version of openstack cloud / ceph we deliver. This is an operational impact.
I would prefer if we were to expose the minimum API version. This is what ppl need to know.

File: docs/reference/versions/public.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Public Cloud
+
+## OpenStack Services
```
Comment (evrardjp) [inline docs/reference/versions/public.md:3]: Same comment as above.

File: docs/reference/versions/index.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+## OpenStack Services
+
+[OpenStack releases](https://releases.openstack.org/) are named, in
+alphabetical order, and occur on a six-month release schedule. In
+{{extra.brand}} Public Cloud we upgrade OpenStack releases annually; this
+means that we deploy every other OpenStack release and skip the
+intervening one.
+
+{{extra.brand}} Cloud currently runs OpenStack
```
Comment (evrardjp) [inline docs/reference/versions/index.md:24]: same comment as above.

File: docs/reference/versions/index.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+* [Public Cloud](public.md): versions running in our {{extra.brand}}
+  Public Cloud regions.
+
+* [Compliant Cloud](compliant.md): versions running in our
+  {{extra.brand}} Compliant Cloud regions.
+
+
+## OpenStack Services
+
+[OpenStack releases](https://releases.openstack.org/) are named, in
```
Comment (evrardjp) [inline docs/reference/versions/index.md:18]: This is very operational. I am not eager to show that in. Except if this is in SLA, which I don't think it is.

**Review by evrardjp — DISMISSED** [review]
Here is my stance on what we should expose to our customers.

File: docs/reference/features/compliant.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+| Virtual GPU   | :material-timer-sand: | :material-timer-sand: |
+
+
+## Block storage
+
+|                                | Sto1HS           | Sto2HS           |
+| ------------------------------ | ---------------- | ---------------- |
+| Highly available storage       | :material-check: | :material-check: |
+| High-performance local storage | :material-close: | :material-close: |
+| Boot from persistent storage   | :material-check: | :material-check: |
```
Comment (noonedeadpunk) [inline docs/reference/features/compliant.md:27]: Um, what is that and how different from HA storage? Like ephemeral vs volumes?

Comment (fghaas) [reply docs/reference/features/compliant.md:27]: "Highly available storage" is ephemeral storage that survives a compute node outage.

"Boot from persistent storage" is boot-from-volume.

Comment (noonedeadpunk) [reply docs/reference/features/compliant.md:27]: Maybe it's just my perception, but it sounds like different things. So I'd understood that persistent storage is not HA and HA is not persistent.
Also saying about boot-from-volume separately makes quite limited sense to me as I actually don't see the way to prohibit it. Well, ofc except overriding policies:)

Comment (fghaas) [reply docs/reference/features/compliant.md:27]: Ok so should I just remove the "boot from persistent storage" row?

Comment (fghaas) [reply docs/reference/features/compliant.md]: Removed in 9a78b693cb91f00d0d5fc35d6fd6fbc75ad29f14.

File: docs/reference/versions/public.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+
+|                                | Kna1      | Sto2   | Fra1  | Dx1   | Tky1   |
```
[the remark this answers is not in the document]
Comment (fghaas) [reply docs/reference/versions/public.md]: Updated in 6a32ee6e171754c4130fdf3ff6e310342b3091f7.

File: docs/reference/versions/compliant.md (hunk 1 above)
Comment (evrardjp) [reply docs/reference/versions/compliant.md:1]: You can ignore my comment above as of now, as the same information is published on our main site.

I am not fond of duplicating the information, as now we have two places to edit. 

I encourage you to remove it once again. Yet, the content itself is not hurting, as of today.

File: docs/reference/versions/public.md (hunk 2 above)
Comment (evrardjp) [reply docs/reference/versions/public.md:3]: I will mark this as resolved, as it's intertwined with my comment above.

File: docs/reference/versions/index.md (hunk 4 above)
Comment (evrardjp) [reply docs/reference/versions/index.md:18]: see also my comment above. Marking this as resolved.

**Review by noonedeadpunk — APPROVED** [review]

**Comment by fghaas** [comment]
@atta Do you have anything else to add/correct on this one?

**Comment by fghaas** [comment]
I guess we won't be getting more additions/corrections on this particular PR. @nerde-push-code, can I ask you to please add an approval on this one so we can merge?

**Review by nerde-push-code — APPROVED** [review]
Ok. Please go ahead.

**Pull request merged**